bokehjs/src/coffee/common/embed.coffee

```diff
@@ -8,6 +8,8 @@ HasProperties = require "./has_properties"
 {pull_session} = require "./client"
 {Promise} = require "es6-promise"
 
+documents = {}
+
```

Comment:
heh, I just realized I suggested this global over on the issue for this PR. Anyway...